webserver/external/messybrainz.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import ujson
+
+def get_messybrainz_id(querydata):
+    """
+    This function queries messybrainz to get a msid according to the 'artist/title' present in the item metadata
+    Args:
+        querydata: A dictionary containing at least artis and song title
+
+    Returns:
+        msid: string messybrainz id -> if it is found in the messybrainz database
+        None: if no reference is fount in messybrainz database
+
+    """
+    msurl = 'https://messybrainz.org/submit'
+    if isinstance(querydata, list):
```

Comment:
Yep, that's true.
However, it's important to consider inputs and outputs of functions and how you are going to be using them. If you think about the flow of how data comes into AcousticBrainz via this method, it can help to make a decision on the best parameters to have.
We have an endpoint which takes a single json file from the extractor. From this, we extract the title and artist, and make a request to messybrainz.
At no point in this workflow do we consider more than one recording at a time (the usecase for having a list of items). Therefore, it would be reasonable to make the `get_messybrainz_id` take as parameters only data for one song.
If the messybrainz API requires input to be a list, that's fine. You can just wrap the data in a list before you make the call:
```python
jsondata = ujson.dumps([querydata])
```
In fact, you already make an assumption that there is only ever one item in this list when you check `if 'artist' in querydata[0].keys()`, so by making the input singular you reduce the amount of logic and code that you need for this test